module/VuFind/src/VuFind/Config/ConfigManager.php

```diff
@@ -128,6 +128,24 @@ public function getConfigObject(string $configPath, bool $forceReload = false, b
         return new Config($this->getConfigArray($configPath, $forceReload, $useLocalConfig));
     }
 
+    /**
+     * Get config in PluginManager style.
+     *
+     * @param string $name    Service name of plugin to retrieve.
+     * @param ?array $options Options to use when creating the instance.
+     *
+     * @return mixed
+     *
+     * @deprecated Use getConfigArray or getConfig instead
```

Comment:
Since it's not deprecated yet, and it's the most direct replacement, maybe we should also mention getConfigObject here:

```suggestion
     * @deprecated Use getConfigArray, getConfigObject or getConfig instead
```